### ISSUE TITLE
Incorrect wallet directory location

### ISSUE THREAD

**Issue by kosecki123** [ethereum-alarm-clock/eac.js#2]
Wallet directory contains scripts that are used by eac.js CLI tool. 

For e.g --createWallet feature uses `1_createWallet.js` and tries to require it from `'../wallet/1_createWallet'` which isn't correct location given current directory structure

```
├── docs
├── src
│   ├── assets
│   ├── build
│   │   └── contracts
│   ├── cli
│   ├── client
│   └── contracts
└── wallet
```

This leads to fatal error and --createWallet script termination

```
How many accounts would you like in your wallet?
> 3
Where would you like to save the encrypted keys? Please provide a valid filename or path.
> ./keys
Please enter a password for the keyfile. Write this down!
> abc
[FATAL] Error: Cannot find module '../wallet/1_createWallet'
```

**Comment by lsaether** [comment]
Hey @kosecki123 👋 

This is something I missed during reorg. The wallet directory should be moved into `src` and the files within should be cleaned up a bit since I was using them as utility scripts during development.

**Comment by lsaether** [comment]
22dc4f3dbaf39747eb3aeda2ac13afdeb23844ba fixes this. The `--createWallet` flag should work now.

**Comment by kosecki123** [comment]
Fixed in 22dc4f3. Closing.
